index.html

```diff
@@ -869,9 +887,66 @@ <h2>
                   </li>
                   <li>If acquiring the position data from the system succeeds:
                     <ol>
-                      <li>Set |position| be [=a new `GeolocationPosition`=]
-                      passing |acquisitionTime| and
-                      |options|.{{PositionOptions/enableHighAccuracy}}.
+                      <li>
+                        <div class="correction" id="c5">
+                          <span class="marker">Candidate Correction:</span>
+                          Each acquired data type is now explicitly defined
+                          with units and standards.
+                          <ol>
+                            <li>
+                              <ins>Let |latitude:double| be a real number of
+                              degrees, in the [[WGS84]] coordinate
+                              system.</ins>
```

Comment:
Yeah, it might be cleaner to put everything in to a struct, then iterate over the names assigning each to the attribute values (or corresponding internal slots, if we use them). I'll check if infra provides a facility to do that. 

In payment request, we got around this by creating a "fake" init dictionary, as it gives us the defaults too.